Missions/__AR/11_cont/AI.src

```diff
@@ -2724,20 +2760,33 @@ Every 0$1+0$0.2 do
           if BaseUnderAttack(base,-1) then
                continue;
 
+          // [SHARK] these should count against reservesCount as well! limit them as well.
+
           //The bazookers to attack arrive
-          special_russian_attackers = CreateUnitsWithClass(difficulty + yashin_alive, class_bazooker, side);
+          if russian_reserve_troops > 0 then
```

Comment:
should these really count towards the same reserve? on one hand - yes. on the other hand - when reserves dry out, gensher is perfectly safe, so his base is overly buffed for that situation.

and since americans still destroy it (apart from him sacrificing his military units on the american base), getting that medal seems impossible. 

... but it wasn't a few changes ago. 🤔